src/windows/mod.rs

```diff
@@ -183,6 +183,48 @@ pub const ENOTEMPTY: ::c_int = 41;
 pub const EILSEQ: ::c_int = 42;
 pub const STRUNCATE: ::c_int = 80;
 
+// POSIX Supplement (from errno.h)
+pub const EADDRINUSE: ::c_int = 100;
```

Comment:
Per the comment below from @UdamSaini , does it make sense to add this one here ? (or maybe we can add it as `#[deprecated]` with the same warning as the windows headers?

Reply:
Those warnings are on the winsock definitions that just redefine to the WSA constants, not the errno definitions.